src/main/java/com/google/devtools/build/lib/actions/CommandLine.java

```diff
@@ -108,9 +102,10 @@ public Iterable<String> arguments() throws CommandLineExpansionException, Interr
     }
 
     @Override
-    public Iterable<String> arguments(ArtifactExpander artifactExpander)
+    public Iterable<String> arguments(ArtifactExpander artifactExpander, PathMapper pathMapper)
         throws CommandLineExpansionException, InterruptedException {
-      return IterablesChain.concat(executableArgs, commandLine.arguments(artifactExpander));
+      return IterablesChain.concat(executableArgs,
+          commandLine.arguments(artifactExpander, pathMapper));
```

Comment:
where is this `pathMapper` is being used to take care of the unstripped arguments?

Reply:
I see pathMapper is being passed in here but it's not being used afterwards?

I think what I meant by unstripped is that with your. changes, at this point all arguments remained unmapped (I'll try to adhere to this term from now on). Is that correct?

Reply:
It is used by overrides of `CommandLine#arguments`, only the default implementation doesn't take this argument into account. For example, this is how `StarlarkCustomCommandLine#arguments` receives the `PathMapper`.